cmd/ping.go

```diff
@@ -2,12 +2,16 @@ package cmd
 
 import (
 	"fmt"
-
+	"github.com/hbagdi/deck/file"
 	"github.com/hbagdi/deck/utils"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
+var (
+	pingCmdKongStateFile []string
+)
+
 // pingCmd represents the ping command
 var pingCmd = &cobra.Command{
 	Use:   "ping",
```

Comment:
The entire purpose of `ping` command is to verify connectivity with Kong. It shouldn't take state file as input. What we can do is instead add in a `--workspace` flag as proposed in #157

Reply:
Ok i should do like the other PR, thank you @hbagdi